### PR TITLE
Dont add "Trending" on gists

### PR DESCRIPTION
**Pull request by jacobbearden**
Noticed that the "Trending" menu item is being added on gists, this fixes it.

### CODE REVIEW

File: extension/content.js [hunk 1]
```diff
@@ -71,6 +71,10 @@ function addReleasesTab() {
 function addTrendingMenuItem() {
 	const $secondListItem = $('.header-nav.left .header-nav-item:nth-child(2)');
 
+	if (pageDetect.isGist()) {
```
Comment (paulmolluzzo) [inline extension/content.js]: Similar to the comments in #228, this doesn't follow the pattern for other page-specific implementations we have. I would take this logic out of the function and then wrap the function call in page detection, which lets this function only be responsible for the DOM updates.

**Comment by paulmolluzzo** [comment]
Left a line comment, but it's easier to just wrap the function call like this:

```js
if (!pageDetect.isGist()){
    addTrendingMenuItem();
}
```

**Comment by paulmolluzzo** [comment]
Also, nice catch and thanks for the PR @jacobbearden!

**Comment by jacobbearden** [comment]
@paulmolluzzo moved it!

**Comment by paulmolluzzo** [comment]
LGTM. 👍

**Pull request merged**